### PR TITLE
Publication Card implementation issue

### PR DESCRIPTION
**Pull request by neelesh-tippe**
publication card implementation issues updated in new branch

### CODE REVIEW

File: stories/Templates/PublicationPage/publicationpage.scss [hunk 1]
```diff
@@ -64,6 +78,10 @@
     margin-bottom: $spacing-24;
   }
 
+  &-36 {
```
Comment (shishirphadke) [inline stories/Templates/PublicationPage/publicationpage.scss:81]: @neelesh-tippe this class is written twice, once on line #48 and line #81. Same goes for `&-40`. Can you write it once?

File: stories/assets/js/global-variables.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+// const BREAKPOINT = window.BREAKPOINT || {};
```
Comment (shishirphadke) [inline stories/assets/js/global-variables.js:1]: @neelesh-tippe remove commented code.

File: stories/Templates/PublicationPage/publicationpage.scss (hunk 1 above)
Comment (neelesh-tippe) [reply stories/Templates/PublicationPage/publicationpage.scss:81]: Those are different classes for margin-top and margin-bottom with same space.

File: stories/assets/js/global-variables.js (hunk 2 above)
Comment (neelesh-tippe) [reply stories/assets/js/global-variables.js:1]: removed

**Pull request merged**